ui/redux/actions/user.js

```diff
@@ -784,3 +839,40 @@ export function doCheckYoutubeTransfer() {
       });
   };
 }
+
+/***
+ * Receives a csv of channel claim ids, hits the backend and returns nicely formatted object with relevant info
+ * @param claimIdCsv
+ * @returns {(function(*): Promise<void>)|*}
+ */
+export function doFetchUserMemberships(claimIdCsv: string) {
+  return async (dispatch) => {
+    // check if users have odysee memberships (premium/premium+)
+    const response = await Lbryio.call('membership', 'check', {
+      channel_id: '80d2590ad04e36fb1d077a9b9e3a8bba76defdf8',
+      claim_ids: claimIdCsv,
+    });
+
+    let updatedResponse = {};
+
+    // loop through returned users
+    for (const user in response) {
```

Comment:
- Adding a constant for `80d2590ad04e36fb1d077a9b9e3a8bba76defdf8` ("ODYSEE_CHAN_ID"?) and documenting why it is that way would be useful.
- `loop through returned users` less useful, since the syntax and variable name is self-documenting -- it's a for-loop of `user`s.